CHANGELOG.md

```diff
@@ -58,6 +58,7 @@ in the naming of release branches.
 
 ### Changed
 
+- Removed the module Test.Cardano.Ledger.Generic.Types. Functionality moved to Test.Cardano.Ledger.Generic.Functions
```

Comment:
It's much easier to read the changelog (or any other markdown doc) when modules, functions types names, etc. are marked as code
```suggestion
- Removed the module `Test.Cardano.Ledger.Generic.Types`. Functionality moved to `Test.Cardano.Ledger.Generic.Functions`
```

Reply:
done